Frameworks/UIKit/UIColor.mm

```diff
@@ -81,6 +87,7 @@ - (NSUInteger)retainCount {
 @Status Interoperable
 */
 - (void)dealloc {
+    [super dealloc];
 }
 
```

Comment:
it was intentionally added.  it's a compiler warning (as error) now

---
In reply to: [95707173](https://github.com/Microsoft/WinObjC/pull/1687#discussion_r95707173) [](ancestors = 95707173)